components/admin/markdown-editor.tsx

```
@@ -0,0 +1,152 @@
+          <Textarea
+            id={id}
+            value={value}
+            onChange={(e) => onChange(e.target.value)}
+            placeholder="Write your content in Markdown..."
+            className="min-h-[300px] border-0 rounded-none focus-visible:ring-0 resize-y"
+          />
+        </TabsContent>
+
+        <TabsContent value="preview" className="p-4 prose dark:prose-invert max-w-none">
+          {value ? (
+            <div dangerouslySetInnerHTML={{ __html: renderMarkdown(value) }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)